featuretools/computational_backends/calculate_feature_matrix.py

```diff
@@ -156,8 +156,8 @@ def calculate_feature_matrix(features, entityset=None, cutoff_time=None, instanc
             raise ValueError(msg)
 
     target_entity = entityset[features[0].entity.id]
-
     cutoff_time = _validate_cutoff_time(cutoff_time, target_entity)
+    entityset._check_time_indexes()
```

Comment:
If I'm thinking about this right, I think the only check that needs to happen here is that all of the entities (eventually woodwork dataframes) have the same time index type. Woodwork will prevent users from setting a time index column if that column does not contain valid numeric or datetime values, so that does not need to be checked again here.

Reply:
✅  cc461753